### ISSUE TITLE
No Voices Available

### ISSUE THREAD

**Issue by rebeltaz** [jneilliii/OctoPrint-M117SpeechSynthesis#10]
Hey... I installed this as per your suggestion on the OctoPrint forum, but when I enabled it, there are no voices available for me to select. I installed it on a OctoPi install on a Raspberry Pi. What do I need to install to get the voices? Thanks.

**Comment by jneilliii** [comment]
The voices load in from your system/ browser. What browser are you using and are there any errors in the developer console (typically press f12 in the browser)

**Comment by rebeltaz** [comment]
f12 does pull up the developer console, but I wouldn't have any idea how to use that.

I tried in in Firefox and I see voices there. My main browser, though, is Brave - which is a Chromium-based browser - running under Ubuntu Linux v18.04 LTS. I tried adding an extension I found called Read Aloud that works with cloud voices, but that didn't seem to have any effect on this plugin. I also tried installing espeak and festival for linux which had no effect, either.
